pypeit/core/arc.py

```diff
@@ -78,6 +72,8 @@ def setup_param(spectro_class, msarc_shape, fitstbl, arc_idx,
                                                      spectro_class.spectrograph,
                                                      wvmnx=arcparam['wvmnx'],
                                                      modify_parse_dict=modify_dict)
```

Comment:
Can you create a doc string for this method?

Reply:
I didn't write this method.